debarshi/js/utils/helperfuncs.js

```diff
@@ -0,0 +1,65 @@
+const validateInput = (value = "") => {
```

Comment:
file naming doesn't seems correct. should be helpers.js/utils.js etc but not helpersfuncs.js since helpersfuns is neither snake case or any other standard naming convention